### PR TITLE
[Consignments] Simplify flow

### PR DESCRIPTION
**Pull request by mennenia**
for https://github.com/artsy/collector-experience/issues/795

simulator screen shot - iphone 6 - 9 1 - 2017-12-15 at 16 27 38
simulator screen shot - iphone 6 - 9 1 - 2017-12-15 at 16 27 49
simulator screen shot - iphone 6 - 9 1 - 2017-12-15 at 16 30 14

@katarinabatina I also updated the spacing on the welcome screen

#skip_new_tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Consignments/Typography/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/mc-consignments/src/lib/Components/Consignments/Screens/__tests__/Overview-tests.tsx">Overview-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/mc-consignments/src/lib/Components/Consignments/Screens/__tests__/Overview-tests.tsx#L126'>adds submitted before triggering final submission on submit</a>
<br/>
Expected mock function to have been called with: [Function anonymous] as argument 2, but it was called with [Function anonymous].

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(jest.fn()).toBeCalledWith(expected)

Expected mock function to have been called with:
  [Function anonymous] as argument 2, but it was called with [Function anonymous].
    at Object.<anonymous> (/home/travis/build/artsy/emission/src/lib/Components/Consignments/Screens/__tests__/Overview-tests.tsx:126:12)
    at Object.asyncFn (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmine_async.js:68:30)
    at resolve (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:47:12)
    at tryCallTwo (/home/travis/build/artsy/emission/node_modules/promise/lib/core.js:45:5)
    at doResolve (/home/travis/build/artsy/emission/node_modules/promise/lib/core.js:200:13)
    at new Promise (/home/travis/build/artsy/emission/node_modules/promise/lib/core.js:66:3)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:34:19)
    at promise.then (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:75:39)
    at tryCallOne (/home/travis/build/artsy/emission/node_modules/promise/lib/core.js:37:12)
    at /home/travis/build/artsy/emission/node_modules/promise/lib/core.js:123:15
</code></pre>
</details>
</li>
<br/>


  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Consignments/Screens/__tests__/Overview-tests.tsx [hunk 1]
```diff
@@ -115,5 +121,10 @@ describe("Updating State", () => {
     expect(stateMock).toBeCalledWith({ provenance: "This is my provenance" }, update)
   })
 
+  it("adds submitted before triggering final submission on submit", () => {
+    overview.goToFinalSubmission()
+    expect(stateMock).toBeCalledWith({ state: "SUBMITTED" }, update)
```
Comment (mennenia) [inline src/lib/Components/Consignments/Screens/__tests__/Overview-tests.tsx]: @orta This test is still failing..

File: src/lib/Components/Consignments/Screens/Overview.tsx [hunk 2]
```diff
@@ -127,6 +132,7 @@ export default class Info extends React.Component<Props, State> {
   }
 
   submitFinalSubmission = async (setup: ConsignmentSetup) => {
+    console.log(this.state)
```
Comment (orta) [inline src/lib/Components/Consignments/Screens/Overview.tsx]: console logs

Comment (orta) [reply src/lib/Components/Consignments/Screens/Overview.tsx]: I shipped a fix

**Pull request merged**